recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "4" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
+{% set build_ext_version = "1.0.0" %}
+{% set build_ext = "cuda" if cuda_enabled else "cpu" %}
+{% set build_string = "h{}_{}".format(PKG_HASH, number) %}
+{% set build_string_ext = "{}_{}".format(build_string, build_ext) %}
+{% set py_build_string_ext = "py{}{}_{}".format(CONDA_PY, build_string, build_ext) %}
+{% set py_build_string = "py{}{}".format(CONDA_PY, build_string) %}
+{% set install_base = "opt/heavyai" %}
+{% set arrow_version = "7.*" %}
+{% set arrow_proc_version = "3.*" %}
+{% set pythrift_version = "0.16.*" %}
+{% set thrift_version = "0.16.*" %}
+# heavydb 6.0.0 is not LLVM 12 ready
+{% set llvm_version = "11" %}
+
+package:
+  name: heavydb-ext
```

Comment:
For multi-output recipes, we usually use `heavydb-split` or similar names. I am not sure what `-ext` means here, but it's not important. Just a matter of taste. What we should really do is to annotate the `extras` field with [`feedstock-name`](https://github.com/conda-forge/qtconsole-feedstock/blob/3b0c1ca81dc202f66e0f08f4968b92a3e695eb27/recipe/meta.yaml#L80) so you obtain a `conda-forge/heavydb-feedstock` repo, and not `conda-forge/heavydb-ext-feedstock`.

Reply:
We used `arrow-cpp` feedstock as an example in introducing `heavydb-ext` because `arrow-cpp` and `heavydb` are in many ways similar: both have CPU and CUDA targets, and python bindings.